composer.json

```diff
@@ -73,6 +73,7 @@
             "OpenEMR\\Entities\\" : "entities",
             "OpenEMR\\Services\\" : "services",
             "OpenEMR\\Repositories\\" : "repositories",
+            "OpenEMR\\Encounter\\Services\\" : "interface/patient_file/encounter/Services",
```

Comment:
pretty sure this is not supposed to be here

Reply:
yes, will remove

Reply:
Done